include/rc_client.h

```diff
@@ -317,6 +317,91 @@ typedef struct rc_client_subset_t {
 
 RC_EXPORT const rc_client_subset_t* RC_CCONV rc_client_get_subset_info(rc_client_t* client, uint32_t subset_id);
 
+/*****************************************************************************\
+| Fetch Game Hashes                                                           |
+\*****************************************************************************/
+
+typedef struct rc_client_hash_library_entry_t
+{
```

Comment:
Opening brace should be on same line as type name (applies to other structs in this file)